larray/viewer.py

```diff
@@ -2176,8 +2176,12 @@ def update_mapping(self, value):
         changed_keys = [k for k in keys_after if value[k] is not self.data.get(k)]
```

Comment:
Indeed, that's a good point/catch. But I do not think there is any practical way to do this. We would need to keeps copies of every array in memory just to check them... :(